lib/services/database/comment_repository_service.dart

```diff
@@ -0,0 +1,124 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/models/database/comment/comment_data.dart";
+import "package:proxima/models/database/comment/comment_firestore.dart";
+import "package:proxima/models/database/comment/comment_id_firestore.dart";
+import "package:proxima/models/database/post/post_data.dart";
+import "package:proxima/models/database/post/post_firestore.dart";
+import "package:proxima/models/database/post/post_id_firestore.dart";
+import "package:proxima/services/database/firestore_service.dart";
+
+/// This class is a service that allows to interact with the comments
+/// of the posts in the firestore database.
+class CommentRepositoryService {
+  final FirebaseFirestore _firestore;
+
+  CommentRepositoryService({required FirebaseFirestore firestore})
```

Comment:
To stay consistent with other Services formatting:

```suggestion
  CommentRepositoryService({
    required FirebaseFirestore firestore,
  }) : _firestore = firestore;
```

Reply:
Fix in 159a7c3c6ee3839739a78bf7df1736f6d97b150b :+1: